babel.config.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
     '@babel/plugin-proposal-object-rest-spread',
     '@babel/plugin-proposal-nullish-coalescing-operator',
     '@babel/plugin-proposal-optional-chaining',
+    'import-graphql',
```

Comment:
This automatically parses a .graphql import into an AST for use. 

It does the same thing as this:

https://github.com/apollographql/graphql-tag